docs/detections/rules-ui-create.asciidoc

```diff
@@ -114,75 +130,169 @@ image::images/rule-query-example.png[]
++
+For example, to create a rule that generates alerts when `host.name` *and*
+`destination.ip` field values in the `logs-*` or `packetbeat-*` {es-sec} indices
+are identical to the corresponding field values in the `mock-threat-list` threat
+index, enter the rule parameters seen in the following image:
++
+[role="screenshot"]
+image::images/indicator-rule-example.png[]
+TIP: When an indicator match rule's conditions are met, the resulting detection alert does not contain explicit information about which event field(s) match which indicator field(s). As such, when you <<rule-ui-basic-params, configure basic rule settings>>, it is recommended that you include a reference to the field(s) to be matched in the rule `Name` and rule `Description`, and ensure that the Timeline template associated with the rule includes pre-defined column(s) for these fields. For example, if you create an indicator match rule that looks for matches between the `file.extension` field in file events and the `threat.file.extension` field in an indicator index, you might name your rule *"file.extension matches ransomware file extension"*, so that when an analyst investigates the detection alerts, they will see the rule name and know to further investigate the `file.extension` field value.
 . Select the Timeline template used when you investigate an alert created by
 the rule in Timeline (optional).
 +
 TIP: Before you create rules, create <<timelines-ui, Timeline templates>> so
 they can be selected here. When alerts generated by the rule are investigated
 in Timeline, Timeline query values are replaced with their corresponding alert
 field values.
-
-. Click *Continue*.
 +
-The *About rule* pane is displayed.
+. Click *Continue*. The *About rule* pane is displayed.
++
 [role="screenshot"]
 image::images/about-rule-pane.png[]
++
+. Continue with <<preview-rules, previewing the rule>> (optional) or <<rule-ui-basic-params, configuring basic rule settings>>.
+
+[discrete]
+[[preview-rules]]
+=== Preview your rule (optional)
+
+You can preview a custom query, threshold, or EQL (Event Correlation) rule to get feedback on how noisy the rule will be before submitting it. This allows you to fine-tune the query, if needed, to reduce the amount of alerts that may come in.
 
-. Continue with <<rule-ui-basic-params>>.
+To preview a rule:
+
+. Write the rule query.
++
+. Select a timeframe of data to preview query results -- `Last hour`, `Last day`, or `Last month` -- from the *Quick query preview* drop-down.
++
+. Click *Preview results*. A histogram shows the number of alerts you can expect based on the defined rule parameters and historical events in your indices.
+
+A "noise warning" is displayed if the preview generates more than alert per hour.
+
+[role="screenshot"]
+image::images/preview-rule.png[]
+
+NOTE: The preview shows a maximum number of 100 results and excludes effects of rule exceptions and timestamp overrides.
```

Comment:
I don't believe the preview is locked to 100 results (at least for the custom query and threshold rule):

##### Custom Query
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/98740047-a4e4e500-2367-11eb-8788-1a88544f6fc1.png" />
</p>

##### Threshold
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/98740267-fb522380-2367-11eb-82b9-9480fa25d666.png" />
</p>

Reply:
Okie, so just verified with @yctercero that the 100 result cap is specific to Event Correlation rules, not the others.